inst/include/dust/r/dust.hpp

```diff
@@ -375,31 +375,31 @@ void dust_set_n_threads(SEXP ptr, int n_threads) {
 }
 
 template <typename T, typename std::enable_if<!std::is_same<dust::no_data, typename T::data_type>::value, int>::type = 0>
-void dust_set_data(SEXP ptr, cpp11::list r_data) {
+void dust_set_data(SEXP ptr, cpp11::list r_data, bool shared) {
```

Comment:
I think I would prefer `shared_data` throughout, as I am thinking of plain `shared` as shared_pars or the shared L1 cache